IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterways/PropertySets/Pset_MaintenanceTriggerPerformance/DocPropertySet.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MaintenanceTriggerPerformance" UniqueId="3b24f630-a25e-4015-af67-291775ce97d3" ApplicableType="IfcElement,IfcAsset,IfcSystem" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
 	<Properties>
-		<DocProperty xsi:nil="true" href="TargetPerformance_3YBpMKO1H0zhLW9cAjhxfY" />
-		<DocProperty xsi:nil="true" href="PerformanceUnit_13zaQi92X5qf27UlqY1Qkg" />
+		<DocProperty xsi:nil="true" href="zzTargetPerformance_3YBpMKO1H0zhLW9cAjhxfY" />
 		<DocProperty xsi:nil="true" href="MaintenanceLevel_1iqiV1Bdv9kP1zFmTnxXvA" />
 		<DocProperty xsi:nil="true" href="ReplacementLevel_0zVRGQxr18AMXYOzl_qyE" />
 		<DocProperty xsi:nil="true" href="DisposalLevel_2uwAVU8D7Mfn2xkuHNa0V" />
+		<DocProperty xsi:nil="true" href="x2vDyZ4jX80fnC8diCnxQ3" />
```

Comment:
Name of property?

Reply:
its the one above you found.

Reply:
as above i removed the property and it has gone funny in IfcDoc...